data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -44,11 +72,25 @@
             villages_per_scout=16
         [/ai]
         side=4
-        team_name=north-west
-        user_team_name= _ "teamname^Northwest"
+        team_name=northwest
+        user_team_name= _ "Northwest"
         controller=human
         gold=150
     [/side]
+    [side]
+        [ai]
+            villages_per_scout=16
+        [/ai]
```

Comment:
I only just noticed this, but why are the `[ai]` blocks at the top of each `[side]` instead of at the bottom? It does look like it was already like that before you started, so not really relevant to this pull request, but I thought we had a convention of always placing keys above tags (with a few exceptions in certain special circumstances, but this wasn't one of those for sure).

Reply:
Good catch, moved them to the bottom.